src/abConfiguration.js

```diff
@@ -56,11 +56,11 @@ ABConfiguration.prototype._getNonTrueVariants = function() {
 };
 
 ABConfiguration.prototype._getSplit = function() {
-  return this._splitRegistry ? this._splitRegistry[this._splitName] : null;
+  return this._splitRegistry.getSplit(this._splitName);
```

Comment:
do we only ever call this when we have already checked id the registry `isUnavailable`?

Reply:
`getSplit` checks if the registry unavailable, so this is safe.